### DISCUSSION TITLE
"What Links Here" - view backlinks

### DISCUSSION THREAD

**Discussion by metasj** [knowledgefutures/pubpub#1377] [Ideas]
Related to #649 but not to #630 - 

As a reader and editor, I'd like to see the list of other pubs that link to a given pub.  (Perhaps also a pingback list that shows other sites online that link to it, at least those that use a feed format supporting pingbacks)

As an editor and community admin, this helps clean up references that may want to change as pubs change; helps see the context of conversation across pubs; and generally visualizes whether a document is in active use beyond its revision and comment history.

As a reader, this helps me follow conversations and see documents as part of a network of information.

**Comment by gabestein** [comment]
Hi everyone — I'm going in and converting some of the longer-lived feature request/idea issues from Issues to Discussion threads. There's no exact heuristic about what belongs in the forum vs. what belongs as an issue, so when in doubt, please post wherever you see fit — we won't mind! But in general, we'd like to make Issues a space for things that we're committed to building/fixing, and Discussions a space for most everything else. So, if you're reporting a bug or requesting we make a straightforward change to an existing feature, go ahead and log an issue. Otherwise, Discussions is likely a better bet.

So, in this case, we're moving it to discussions and inviting anyone who's also interested in the idea to comment!

This one is really interesting, and definitely something we want to explore as part of the Impact section. We also think there's likely a larger need for this type of "open altmetrics" functionality throughout the community, and we'd love to explore it with other developers who are interested in similar ideas/features.
